addons/coupon/wizard/coupon_generate.py

```diff
@@ -4,9 +4,9 @@
 from odoo.tools.safe_eval import safe_eval
 
 
-class SaleCouponGenerate(models.TransientModel):
-    _name = 'sale.coupon.generate'
-    _description = 'Generate Sales Coupon'
+class CouponGenerate(models.TransientModel):
+    _name = 'coupon.coupon.generate'
```

Comment:
Nitpicking: redundant name is redundant; for me, `coupon.generate` is enough; `coupon.generate.wizard` if you prefer